tests/acs/test_unit.py

```diff
@@ -219,6 +219,7 @@ def test_lanczos_with_grid(self):
         assert(med_diff < 1.0e-6)
         assert(max_diff < 1.0e-5)
 
+    pytest.mark.skip(reason='Point kernel no longer supported')
```

Comment:
The text here should be "Tophat" instead of "Point".

Reply:
fixed.